docs/content/events/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Events"
+heading: NIST OSCAL Community Events Calendar
+aliases:
+  - /events/
+---
+
+## Calendar
+
+The NIST OSCAL Team has a public calendar for the community to keep current with the meetings above and other public events hosted by the team.
+
+You can access the [web version of this calendar by clicking this link](https://outlook.office365.com/calendar/published/553a60383771471381091ed979a954f6@nist.gov/e72d2ca648dc4c3db7470fb7339674f211929765598025548039/calendar.html) or configure your calendar appplication [with this iCalendar (ICS) file](https://outlook.office365.com/owa/calendar/553a60383771471381091ed979a954f6@nist.gov/e72d2ca648dc4c3db7470fb7339674f211929765598025548039/calendar.ics).
```

Comment:
I downloaded the .ics file provided here and it points to ONE OSCAL mini workshop event that occurred on Feb 1st. Also, It lists MS Teams... NOT BJ.

Reply:
UPDATE: Once the calendar event is modified, the ics file does get updated BUT I am still getting only the first event of the month through the '.ics' approach, which is already history. Will continue testing.

Reply:
I loaded it on multiple clients (I do not even use Office365 and Google Calendar, I use a different email system for personal use cases) and I see 3 events none with a MS Teams link. Perhaps it an issue with your setup and we should look at it together?

It is a calendar all the same, it will show historical and future events as they are scheduled on the calendar, and your application must sync. So let's look at the application, but others have reported in casual testing it works.

Reply:
> I loaded it on multiple clients (I do not even use Office365 and Google Calendar, I use a different email system for personal use cases) and I see 3 events none with a MS Teams link. Perhaps it an issue with your setup and we should look at it together?
> 
> It is a calendar all the same, it will show historical and future events as they are scheduled on the calendar, and your application must sync. So let's look at the application, but others have reported in casual testing it works.

The iCalendar (.ics file) gets loaded to the system's (Mac's) Calendar and ALL the events  (past and future) are placed and get updated when updates exist, but I cannot be selective with the events I want to attend /place on my calendar. I get all past and all future ones and I need to go manually and delete them one by one. If this is the expected functionality, then it works as designed.

Reply:
That would be the idea, you can choose to copy events to your own calendar or you accept the invite by email, per the updated information put in the ADR to clarify your questions abut this in usnistgov/OSCAL#1638. I would direct you to look at it. You can choose one or both methods, but if you configure the ICS file in your email client, you do not receive notifications unless you configure your client to do so. I actually have had the calendar running in multiple clients for two weeks (Office365, Google, and other): I can toggle this calendar on and off if I do not want to see it. I only receive notifications on the one which we added to oscal-dev@list.nist.gov. If you or I do not do that, people will not receive any email at all.

I will reiterate why this very small change makes a difference. As it stands, often people need a resource to look and know: "Is a meeting happening in a few minutes or already started?" because we have not coordinate communications across our website and emails (which people can fall behind on and not accept, so historically we have continued to facilitate this problem by only communicating it that way). If you are not on the mailing list and did not save in the moment, you will not easily find this information. Like this morning, we continue to get routine questions in our main chat room about outdated email invites that historically you, Dave, or other members sent.